src/System.Management.Automation/engine/PropertyAccessor.cs

```diff
@@ -681,6 +697,15 @@ internal override void SetDefaultSourcePath(string defaultPath)
         }
 
         /// <summary>
+        /// Allow loading assemblies by searching in GAC.
+        /// </summary>
+        /// <returns>Always true, as this is for Full CLR.</returns>
```

Comment:
I had two options.. 
1. Add it to the interface, hence need to implement it in RegistryAccessor and JsonConfigFileAccessor.
2. Implement it as a static method on JsonConfigFileAccessor and break the strategy pattern.

I ended up going with number 1.